Development.md

```diff
@@ -91,9 +97,11 @@ PASSWORD=45e0f109-3f56-4b30-9b5a-b0939b8a4c25
 ...
 ```
 
+These secrets can be used in your local rack docker vm for interfacing with AWS.
```

Comment:
no clue if this is true or not. I don't know and couldn't find out how this info gets passed into ``convox start``.